packages/synapse-sdk/src/telemetry/singleton.ts

```diff
@@ -0,0 +1,262 @@
+/**
+ * Telemetry singleton manager.
+ * Sets up and provides a single global TelemetryService instance.
+ * #initGlobalTelemetry is the entry point.
+ * #getGlobalTelemetry is the expected access point within Synapse and beyond.
```

Comment:
```suggestion
 * #getGlobalTelemetry is the expected access point within Synapse
```

we want consumers to access it (from outside synapse) with `synapse.telemetry` and should likely do similar if they have access to the synapse instance, instead of calling `getGlobalTelemetry`